### ISSUE TITLE
Whitespaces in Paths

### ISSUE THREAD

**Issue by kritzi** [ardevd/gitwrap#7]
You have to escape whitspaces with "\ " in unix paths:

```
diff --git a/GitWrap/GitWrap/Program.cs b/GitWrap/GitWrap/Program.cs
index 9015502..56666a7 100644
--- a/GitWrap/GitWrap/Program.cs
+++ b/GitWrap/GitWrap/Program.cs
@@ -45,6 +45,8 @@ namespace GitWrap

                 // Convert Windows path to Linux style paths
                 argstr = argstr.Replace("\\", "/");
+                // Escape Whitespaces^M
+                argstr = argstr.Replace(" ", "\\ ");^M
                 argsString += " " + argstr;
             }
```

**Comment by ardevd** [comment]
Thank you very much for reporting the issue. Good catch!

**Comment by ardevd** [comment]
Fixed with fb321a3888d23524dc8186e07e858877546d4bef
